src/sentinel.c

```diff
@@ -137,6 +137,10 @@ static mstime_t sentinel_default_failover_timeout = 60*3*1000;
 #define SENTINEL_SIMFAILURE_CRASH_AFTER_ELECTION (1<<0)
 #define SENTINEL_SIMFAILURE_CRASH_AFTER_PROMOTION (1<<1)
 
+static inline int shouldDisplayParam(dict *params, const char *param) {
+	return params == NULL || dictFind(params,param) != NULL;
```

Comment:
```suggestion
    return params == NULL || dictFind(params,param) != NULL;
```

Reply:
It looks this part code already is updated, please take a look, Thanks